Assets/Mirror/Components/NetworkProximityChecker.cs

```diff
@@ -76,6 +81,17 @@ public override bool OnRebuildObservers(HashSet<NetworkConnection> observers, bo
             // otherwise add everyone in proximity
             else
             {
+                // always add self as observer and continue if desired to always keep local player observer
+                if (alwaysKeepLocalPlayerObserver)
+                {
+                    // ensure player can still see themself
+                    var uv = GetComponent<NetworkIdentity>();
+                    if (uv.connectionToClient != null)
```

Comment:
The variable uv is unnecessary. Just access connectionToClient directly, it is inherited from NetworkBehaviour.